requirements.txt

```diff
@@ -12,6 +12,7 @@ django-model-utils==4.5.1
 django-sitetree==1.17.2
 djangorestframework==3.15.2
 djangorestframework-gis==1.0
+djangorestframework-simplejwt==5.5.1
```

Comment:
Adding `django-cors-headers` wouldn't be bad either.

ref: [`g3w-admin-authjwt/requirements.txt`](https://github.com/g3w-suite/g3w-admin-authjwt/blob/35077a4a29fe7cd115622479ce798454b942a909/requirements.txt#L1)